apps/src/lib/ui/accounts/ManageLinkedAccountsController.js

```diff
@@ -1,12 +1,51 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import $ from 'jquery';
+import {navigateToHref} from '@cdo/apps/utils';
 import ManageLinkedAccounts from './ManageLinkedAccounts';
 
 export default class ManageLinkedAccountsController {
-  constructor(mountPoint) {
+  constructor(mountPoint, userType, authenticationOptions) {
+    this.mountPoint = mountPoint;
+    this.userType = userType;
+    this.authenticationOptions = authenticationOptions;
+    this.renderManageLinkedAccounts();
+  }
+
+  renderManageLinkedAccounts = () => {
     ReactDOM.render(
-      <ManageLinkedAccounts/>,
-      mountPoint
+      <ManageLinkedAccounts
+        userType={this.userType}
+        authenticationOptions={this.authenticationOptions}
+        connect={this.connect}
+        disconnect={this.disconnect}
+      />,
+      this.mountPoint
     );
-  }
+  };
+
+  connect = (provider) => {
+    navigateToHref(`/users/auth/${provider}/connect`);
+  };
+
+  disconnect = (authOptionId) => {
```

Comment:
I feel like this method could be a lot simpler, either using fetch or async/await or both.